identity/app/model/EmailSubscriptions.scala

```diff
@@ -97,6 +97,7 @@ object EmailSubscriptions {
       frequency = "Tuesday mornings",
       listId = "3698",
       subscribedTo = subscribedListIds.exists{ x => x == "3698" },
+      signupPage = Some("https://www.theguardian.com/politics/2016/may/31/eu-referendum-morning-briefing-sign-up"),
```

Comment:
Not sure if we can have a relative path? Email prefs is on https://profile.theguardian.com/email-prefs so LinkTo would try and take the user to `https://profile.thegulocal.com/<custom-path>/latest/email`